include/z64save.h

```diff
@@ -189,4 +186,14 @@ typedef enum {
     /* 0xFF */ BTN_DISABLED = 0xFF
 } ButtonStatus;
 
+typedef enum {
+    /* 0x00 */ HIGHSCORE_HORSE_ARCHERY,
```

Comment:
I'm a bit concerned that this might not be immediately obvious, even with the context that it's a high score, but I do see the motivation for this. Maybe you could expand a little more on why this could be preferred?

Reply:
There are places where we really can't afford the long version, so "hba" will probably remain in use there. And I think we should be consistent with the terms we use, so I suggest using it here as well. But you can also add a more descriptive comment if you think it's necessary (same for any other entries)

Reply:
Makes sense, thanks. I added a comment for now but it can always be removed later if there's no possible confusion.